pkg/placement/selectors.go

```diff
@@ -72,3 +77,97 @@ func GetKeys(m map[string]string) []string {
 	}
 	return keys
 }
+
+// when an object is updated, we iterate over all placements and update placementToResolutionMap that
+// are affected by the update. Every affected placement decision is then queued to by synced.
+// TODO: when PlacementDecision API supports singleton statuses, this function should be updated.
+func (c *Controller) updateDecisions(obj runtime.Object) error {
+	placements, err := c.listPlacements()
+	if err != nil {
+		return err
+	}
+
+	objMR := obj.(mrObject)
+
+	changedResolutions := map[string]interface{}{}
```

Comment:
I suggest using `Set[string]` here.

Reply:
A slice of strings now suffices since placement-decisions can be queued by name, with no need to distinguish between existing and none-existing ones.